### PR TITLE
Merge the two draft introductions

### PR DESCRIPTION
**Pull request by cgreene**
The two draft introductions needed to be merged. One was more technical that was probably ideal for this review, and the other was a bit colloquial. With this, I tried to strike a balance. The goal of this is to get _something_ in place that is reasonable, so that we can refine next week for fine tuning.

### CODE REVIEW

File: sections/02_intro.md [hunk 1]
```diff
@@ -25,164 +25,46 @@ could learn to identify cats simply by watching online videos
...
+suited to fitting the many parameters that exist for deep neural networks.
+
+The convergence of these factors currently makes deep learning extremely
+adaptable, and capable of addressing the nuanced differences of each domain to
 which it is applied. Many of the advances in deep learning were first developed
-for image analysis and text analysis, but the lessons and techniques learnt
-there enable the construction of very powerful models specifically suited to the
-challenges presented by  each unique domain.
-
-### Deep learning in scientific inquiry
```
Comment (cgreene) [inline sections/02_intro.md]: I am supportive of this. About to push another commit that may address the rest of your points.

File: sections/02_intro.md [hunk 2]
```diff
@@ -25,164 +25,46 @@ could learn to identify cats simply by watching online videos
...
-optimism is justified.
-
-None of these concerns should dampen enthusiasm about deep learning. Rather,
-because the approaches offer such great flexibility, there is good reason to
-believe that carefully defined networks might enable important scientific
-advances.
-
-Another version:
-
-Deep learning is built on a biologically-inspired approach from machine learning
```
Comment (cgreene) [inline sections/02_intro.md]: I was a bit worried that this was too much in the weeds too quickly. A box would be ideal. I briefly mention autoencoders and adversarial networks + unsupervised learning now. I tried to make it skimable if someone doesn't know what they are, but maybe we should add a ref.

File: sections/02_intro.md [hunk 3]
```diff
@@ -25,164 +25,46 @@ could learn to identify cats simply by watching online videos
...
-interpretability that better matches their use case; 2) the available data are
-insufficient to support deeper architectures, however new datasets that will
-support deep methods are expected; 3) or as building blocks to be combined with
-other non-neural-network-based approaches at subsequent stages.
+for image analysis and text analysis, but the lessons learnt and techniques
+developed there can enable the construction of very powerful models specifically
+suited to the challenges presented by many domains.  Neural networks,
+particularly deep ones, are a highly flexible representation capable of
+capturing many different types of features. Deep neural network approaches have
+demonstrated success from the game of Go [@doi:10.1038/nature16961] to quantum
```
Comment (cgreene) [inline sections/02_intro.md]: I removed it from the previous paragraph due to that being otherwise about images.

File: sections/02_intro.md [hunk 4]
```diff
@@ -25,164 +25,46 @@ could learn to identify cats simply by watching online videos
...
-contrastive error has proven particularly useful in modeling language.
-Reinforcement learning has enabled neural networks to learn how to play games
-like chess, GO, and poker. Curriculum learning enables networks to gradually
-build up expertise to solve particularly challenging algorithmic problems.
-Dropout nodes and layers make networks much more robust, even when the number of
-weights are dramatically increased. `TODO: jargon`
-
-Fourth, the convergence of these factors currently makes deep learning extremely
-adaptable, and capable  of addressing the nuanced differences of each domain to
+possible. Easy-to-use software packages including Theano [@arxiv:1605.02688],
```
Comment (cgreene) [inline sections/02_intro.md]: Removed.

File: sections/02_intro.md [hunk 5]
```diff
@@ -25,164 +25,46 @@ could learn to identify cats simply by watching online videos
...
-our scope. But these algorithms can train networks in domains where earlier
-algorithms struggled.  For example, newer optimizers can very efficiently learn
-using batched training, where only a portion of the data needs to be processed
-at a time. These optimizers more effectively optimize very large weight vectors
-where many weights are only rarely updated. `TODO: awkward phrasing` Noise
-contrastive error has proven particularly useful in modeling language.
-Reinforcement learning has enabled neural networks to learn how to play games
-like chess, GO, and poker. Curriculum learning enables networks to gradually
-build up expertise to solve particularly challenging algorithmic problems.
-Dropout nodes and layers make networks much more robust, even when the number of
```
Comment (cgreene) [inline sections/02_intro.md]: I tried to work in dropout + a discussion of a couple types of networks.

File: sections/02_intro.md [hunk 6]
```diff
@@ -25,164 +25,46 @@ could learn to identify cats simply by watching online videos
 What if, more generally, deep learning could solve the challenges
 presented by the growth of data in biomedicine? Could these algorithms
 identify the "cats" hidden in our data - the patterns unknown to the
-researcher - and act on them? Deep learning has transformed image analysis, but
-what about biomedicine more broadly? In this review,
+researcher - and act on them? In this review,
 we examine whether this transformation is simply a matter of time or
```
Comment (cgreene) [inline sections/02_intro.md]: I tried to edit for this, but we can also bring back the sentence. Let me know what you think about the new version.

File: sections/02_intro.md (hunk 2 above)
Comment (agitter) [inline sections/02_intro.md]: Adding a good reference for different network architectures would be helpful.  I recommend we use http://www.deeplearningbook.org/.  The [table of contents](http://www.deeplearningbook.org/contents/TOC.html) shows that it covers every model I would want to introduce.

I like your edit and agree we don't want to go too deep beyond that.

File: sections/02_intro.md (hunk 1 above)
Comment (agitter) [inline sections/02_intro.md]: Here is a candidate new paragraph that is primarily stitched together from the cut text.  I'm not sure that it warrants a new header, but it doesn't fit well with the previous or following header.  @enricoferrero may have comments on how this fits in with the "human factor" now raised in the conclusion.

### Deep learning in biomedical inquiry
> In biomedical contexts, there are domain-specific considerations that influence how to best harness the power and flexibility of deep learning.  Model interpretability can be critical.  Understanding the patterns in data may be just as important as fitting the data.  In addition, there are important and pressing questions about how to build networks that can efficiently represent the underlying structure and logic of the data.  Domain experts can play important roles in designing networks to represent data appropriately, encoding the most salient prior knowledge, and assessing success or failure.  There is also great potential to create deep learning systems that are not intended to replace biologists and clinicians but rather cooperate with them, working to prioritize experiments or streamline tasks that do not require expert judgment.

**Comment by agitter** [comment]
The new changes look good.  I left two new comments about a network architecture reference and paragraph on domain-specific considerations.  See what you think about those, and then merge when you're ready.

File: sections/02_intro.md (hunk 2 above)
Comment (cgreene) [inline sections/02_intro.md]: I can't find the DOI for this. Using a URL citation for now & we can fix later.

Comment (agitter) [inline sections/02_intro.md]: I can't find it either.  The ISBN is 9780262337434.

Comment (cgreene) [inline sections/02_intro.md]: I filled #387 to track this.

**Pull request merged**